SU-SWS/stanford_manage_content
# Basic 2344

# READY FOR REVIEW

# Summary
- Fixes a bug where nodes get deleted that should not
- More information here: https://www.drupal.org/node/2177817#comment-11942691

# Needed By
- ASAP

# Urgency
- Extremely urgent

# Steps to Test

1. Go to `/admin/manage`
2. Check the checkboxes of some nodes
3. Choose "Delete item" from the VBO dropdown
4. Click "Execute"
5. See that the node titles in the confirmation screen do not match the nodes you selected
6. For optional hilarity, click "Confirm" and then run cron to see some random nodes get deleted
7. Check out this branch
8. Repeat steps #2-4
9. See that the node titles in the confirmation screen *do* match the nodes you selected
10. Click "Confirm"
11. See that your node is deleted immediately

# Affected Projects or Products
- All JS sites. I am pulling a list

# Associated Issues and/or People
- [JIRA ticket BASIC-2344](https://stanfordits.atlassian.net/browse/BASIC-2344)
- https://www.drupal.org/node/2177817#comment-11942691
